app-backend/api/src/main/scala/uploads/package.scala

```diff
@@ -12,7 +12,7 @@ package object uploads {
             .filter { p =>
                 val _p = p.toLowerCase
                 _p.endsWith(".tif") ||
-                _p.endsWith(".tif")
+                _p.endsWith(".tiff")
```

Comment:
If that is in preparation for processing the `jpg`s with world files then we I think we would need the world files too.